recommended-setup.js

```diff
@@ -114,6 +121,67 @@ function setupRecommendedScssAndJsTasksAndCopyAssets(
     return this;
 }
 
+function setupRecommended(options) {
+    const config = {
+        assets: [],
+        styles: false,
+        scripts: false,
+        ...options,
+    };
+
+    const build = [];
+    const clean = [];
+    function add(task, buildName = 'build') {
+        build.push(`${buildName}:${task}`);
+        clean.push(`clean:${task}`);
+    }
+
+    if (config.styles) {
+        const styles = config.styles === true ? {} : config.styles;
+        setupRecommendedScssTasks(styles.distBasePath, styles.assetsBasePath, styles.includePaths);
+        add('styles');
+    }
+
+    if (config.scripts) {
+        const scripts = config.scripts === true ? {} : config.scripts;
+        setupRecommendedJsTasks(scripts.distBasePath, scripts.assetsBasePath);
+        add('scripts');
+    }
+
+    if (Array.isArray(config.assets)) {
+        setupCopyAssets(config.assets);
+        add('assets', 'copy');
+    }
+
+    gulp.task('default', gulp.parallel(...build));
+    gulp.task('clean', gulp.parallel(...clean));
+
+    return this;
+}
+
+function getNugetPath() {
```

Comment:
@DAud-IcI What exactly did you need this for again? I don't quite grasp all of what you've added in this PR.

Reply:
It's for getting the physical location of a nuget package in the file system. Similar to the [package path property in MSBuilld](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#generatepathproperty). It was necessary to get packaged assets from the NuGet package and pass it to gulp, e.g. for resolving SCSS files in Lombiq.BaseTheme, see [here](https://github.com/Lombiq/Orchard-Base-Theme/blob/7723404db46cce753675ad9137d503d0556f9e6f/Lombiq.BaseTheme.Samples/Gulpfile.js#L33-L37))

Reply:
Why do we need to know the path of a NuGet package for SCSS compilation? What's the use case?

Reply:
If you want to make a theme utilizing `Lombiq.BaseTheme` as a NuGet package. the localtion of the base theme's scss files need to be told to the sass compiler.

Reply:
I see.